core/api/transport/impl/http_session.hpp

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_API_TRANSPORT_BEAST_HTTP_SESSION_HPP
+#define KAGOME_CORE_API_TRANSPORT_BEAST_HTTP_SESSION_HPP
+
+#include <chrono>
+#include <cstdlib>
+#include <memory>
+
+#include <boost/beast.hpp>
+#include <boost/optional.hpp>
+#include "api/transport/session.hpp"
+
+namespace kagome::api {
+  /**
+   * @brief HTTP session for api service
+   */
+  class HttpSession : public std::enable_shared_from_this<HttpSession>,
+                      public Session {
+   public:
+    struct Configuration {
+      static constexpr size_t kDefaultRequestSize = 10000u;
+      static constexpr Duration kDefaultTimeout = std::chrono::seconds(30);
+      size_t max_request_size{kDefaultRequestSize};
```

Comment:
```suggestion

      size_t max_request_size{kDefaultRequestSize};
```
Just for the sake of readability